IntegrationTests/tests_04_performance/test_01_resources/test_1000_autoReadGetAndSet_sync.swift

```diff
@@ -0,0 +1,43 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2021 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIO
+
+func run(identifier: String) {
+    measure(identifier: identifier) {
+        let iterations = 1000
+        let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+        defer {
+            try! group.syncShutdownGracefully()
+        }
+
+        let server = try! ServerBootstrap(group: group)
+          .bind(host: "127.0.0.1", port: 0)
+          .wait()
+        defer {
+            try! server.close().wait()
+        }
```

Comment:
more nits:

- same here, we could pull most of the code out of the `measure` block.
- instead of spinning up `MTELG`, we could also use `MTELG.withCurrentThreadAsEventLoop`, then you won't need an execute.

Reply:
genius, I always forget about `withCurrentThreadAsEventLoop`!